pseudoanonymize/anonymization.py

```diff
@@ -0,0 +1,92 @@
+from typing import Any, Dict
+
+from .base import BaseProcessor
+from .common import make_chat_completion
+from .exceptions import UnparsableLLMOutputException
+
+with open('./prompts_examples/cot_example.txt', 'r') as file:
+    cot_demo = file.read()
```

Comment:
As a general rule don't use relative paths for production code because they can easily break if someone runs the code from a different working directory.

The way I'd do this is create a file in the root of the repo called `definitions.py` and do the following in there:

```
from pathlib import Path

ROOT_DIR = Path(__file__).parent.absolute()
```

and then in this file do:

```
from definitions import ROOT_DIR

with open(ROOT_DIR / "prompt_examples/cot_example.txt", "r")...
```

Reply:
I removed the prompts_examples folder and inserted the .txt file content into the code. The reason it was set up this way was because it made it easier for me to experiment with different demos when I was iterating over the prompt.

Reply:
I will keep the path tip in mind if I even need to read something from a path. Thanks!